### PR TITLE
Add bulk delete for presentations

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Add bulk delete for presentations

[stage-2]

## Motivation and Context
Network Management epic.

## How Has This Been Tested?
Tested changes locally, updated unit tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Still need to implement a custom error message for Presentations that could not be deleted because they are in a Schedule - in a subsequent PR.

@ezequielc please review. Thanks!

File: web/scripts/editor/controllers/ctr-presentation-list.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ angular.module('risevision.editor.controllers')
   .value('PRESENTATION_SEARCH', {
     filter: ''
   })
-  .controller('PresentationListController', ['$scope',
+  .controller('PresentationListController', ['$q', '$scope',
     'ScrollingListService', 'presentation', 'editorFactory', 'templateEditorFactory', '$loading',
     '$filter', 'presentationUtils', 'PRESENTATION_SEARCH',
-    function ($scope, ScrollingListService, presentation, editorFactory, templateEditorFactory,
+    function ($q, $scope, ScrollingListService, presentation, editorFactory, templateEditorFactory,
```
Comment (ezequielc) [inline web/scripts/editor/controllers/ctr-presentation-list.js:9]: @alex-deaconu `$q` was added but never used. Thanks!

File: web/partials/editor/presentation-list.html [hunk 2]
```diff
@@ -25,15 +25,22 @@ <h1 class="app-header-title" id="title" translate>editor-app.list.title</h1>
...
     <div class="scrollable-list horizontal-scroll border-container u_margin-md-top u_margin-md-bottom"
       scrolling-list="presentations.load()" rv-spinner rv-spinner-key="presentation-list-loader" rv-spinner-start-active="1">
 
       <table id="presentationListTable" class="table">
         <thead class="table-header">
           <tr class="table-header__row u_clickable">
-            <th id="tableHeaderName" class="table-header__cell col-sm-8" ng-click="presentations.sortBy('name')">
-              {{'editor-app.list.heading.name' | translate}}
-              <i ng-if="search.sortBy === 'name'" class="fa" ng-class="{false: 'fa-caret-up', true: 'fa-caret-down'}[search.reverse]"></i>
+            <th id="tableHeaderName" class="table-header__cell col-sm-8">
```
Comment (ezequielc) [inline web/partials/editor/presentation-list.html:36]: @alex-deaconu Your turn to have two `tableHeaderName` :) In th here and div below. Thanks

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍  Minor comments inline, that can be addressed later. Thanks

File: web/scripts/editor/controllers/ctr-presentation-list.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/editor/controllers/ctr-presentation-list.js:9]: @ezequielc good catch, will remove. Thanks.

File: web/partials/editor/presentation-list.html (hunk 2 above)
Comment (alex-deaconu) [reply web/partials/editor/presentation-list.html:36]: @ezequielc 😄 will fix. Thanks!

**Pull request merged**